apps/web/src/components/CommunityInterestDialog/CommunityInterestDialog.tsx

```diff
@@ -0,0 +1,278 @@
+                      trueLabel={intl.formatMessage({
+                        defaultMessage: "Interested in",
+                        id: "AQiPuW",
+                        description:
+                          "Label for user expressing interest in a specific work stream",
+                      })}
+                      falseLabel={intl.formatMessage({
+                        defaultMessage: "Not interested in",
+                        id: "KyLikL",
+                        description:
+                          "Label for user expressing they are not interested in a specific work stream",
+                      })}
+                    >
+                      {workStream.name?.localized ?? notAvailable}
+                    </BoolCheckIcon>
+                  </li>
+                ))}
+              </ul>
+            </>
+          )}
+          <Separator orientation="horizontal" decorative space="sm" />
+          {communityDevelopmentPrograms.length > 0 && (
+            <>
+              <p
+                data-h2-font-weight="base(700)"
+                data-h2-margin-bottom="base(x.25)"
+              >
+                {intl.formatMessage({
+                  defaultMessage:
+                    "Leadership and professional development options",
+                  id: "UfwS5s",
+                  description:
+                    "Label for users interest in development programs for a community",
+                })}
+              </p>
+              <ul
+                data-h2-list-style="base(none)"
+                data-h2-padding-left="base(0)"
+              >
+                {communityDevelopmentPrograms.map((developmentProgram) => {
+                  const interestedProgram =
+                    communityInterest?.interestInDevelopmentPrograms?.find(
+                      (interest) =>
+                        interest?.developmentProgram?.id ===
+                        developmentProgram.id,
+                    );
+                  return (
+                    <DevelopmentProgramInterestItem
+                      key={developmentProgram.id}
+                      developmentProgramInterestQuery={interestedProgram}
+                      label={
+                        developmentProgram?.name?.localized ?? notAvailable
+                      }
+                    />
+                  );
+                })}
+              </ul>
+              <Separator orientation="horizontal" decorative space="sm" />
+            </>
+          )}
+          <p data-h2-font-weight="base(700)" data-h2-margin-bottom="base(x.25)">
+            {intl.formatMessage({
+              defaultMessage: "Additional information",
+              id: "NCMG9w",
+              description:
+                "Label for a community interests additional information",
+            })}
+          </p>
+          <p>{communityInterest?.additionalInformation ?? notAvailable}</p>
```

Comment:
I think the intent might be to only render this section if it is present? Might need to double check with designers
@JoshBeveridge 

![image](https://github.com/user-attachments/assets/4196d4b8-1a82-4beb-840c-60c2fc59f0e3)

Currently. if null 

![image](https://github.com/user-attachments/assets/6e77c408-28af-46d4-b282-42ca2e1a3eae)

Reply:
Woops, missed this one - I think it's safe to omit the section entirely in this case, because the concept behind additional info is that you're optionally supplying it anyway. The omission should be an adequate indicator.

Reply:
Additional information removed from DOM if it was not provided.

[hide additional information if not present](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12524/commits/c3a011c96e15cb289048a4b5bcae6a08b6ea77ad)